www/routes/raw.ts

```diff
@@ -2,13 +2,22 @@ import { RouteConfig } from "$fresh/server.ts";
 import { Handlers } from "$fresh/server.ts";
 import { parse } from "$std/semver/mod.ts";
 import VERSIONS from "../../versions.json" assert { type: "json" };
+import { extname } from "$std/path/mod.ts";
 
 const BASE_URL = "https://raw.githubusercontent.com/denoland/fresh/";
 
+const contentTypes = new Map([
+  [".html", "text/plain"],
+  [".ts", "application/typescript"],
+  [".js", "application/javascript"],
+  [".tsx", "text/tsx"],
+  [".jsx", "text/jsx"],
+  [".json", "application/json"],
+  [".wasm", "application/wasm"],
+]);
+
 export const handler: Handlers = {
-  async GET(req, ctx) {
-    const accept = req.headers.get("Accept");
-    const isHTML = accept?.includes("text/html");
```

Comment:
Let's keep this bit here. It's used to install fresh. There can be multiple values in the `Accept` header which I think the code was trying to account for before.

Reply:
Done!